app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -24,6 +25,24 @@ angular.module("openshiftConsole")
           label: "PiB"
         }];
         scope.claim.selectedLabels = [];
-        }
+//        DataService.list({group: 'storage.k8s.io', resource: 'storageclasses'}, {}, function(storageClasses) {
+//          scope.storageClasses = storageClasses.by('metadata.name');
+//        });
+        DataService.get({group: 'storage.k8s.io', resource: 'storageclasses'}, null, {}, { errorNotification: false})
```

Comment:
I think DataService.list needs to pass opts to `this._startListOp(resource, context, opts);`